src/quick-tour/increase-average-order-value.md

```diff
@@ -4,28 +4,28 @@ title: Increase Average Order Value
 
 Magento provides a range of tools to help you tailor the shopping experience, and encourage your customers to put more items in their shopping carts and spend more money.
 
-[Targeted Promotions]({{ site.baseurl }}{% link marketing/promotions.md %})
-:  Use catalog and shopping cart price rules to create promotions that kick into gear when a set of conditions is met. {% if "Default.EE-B2B" contains site.edition %}Segment customers dynamically and build segments based on specific characteristics such as customer address, order history, shopping cart content, and much more. Then, use [Page Builder]({{ site.baseurl }}{% link cms/page-builder.md %}) to create a [dynamic block]({{ site.baseurl }}{% link cms/dynamic-blocks.md %}) that is triggered by a promotion, and appears only to the targeted customer segment.{% endif %}
+Targeted Promotions
```

Comment:
Same as my previous remarks. Would be nice to have these as headings so they're available in the right-hand TOC.